OpenMarket/OpenMarket/Model/Network/DataEncoder.swift

```diff
@@ -0,0 +1,68 @@
+//
+//  DataEncoder.swift
+//  OpenMarket
+//
+//  Created by 우롱차 on 2022/06/08.
+//
+
+import UIKit
+
+struct DataEncoder {
+
+    private let imageEncoder: ImageEncoder
+    private let jsonEncoder: JSONEncoder
+    private var boundary: String
+    private var boundaryPrefix: String {
+        get {
+            return "\r\n--\(boundary)\r\n"
+        }
+    }
```

Comment:
```suggestion
    private var boundaryPrefix: String {
        return "\r\n--\(boundary)\r\n"
    }
```
`get`은 생략할 수 있을 것 같아요.

Reply:
앗! 수정하도록 하겠습니다.